schema/cassandra/temporal/schema.cql

```diff
@@ -27,6 +27,8 @@ CREATE TABLE executions (
   timer_encoding                 text,
   visibility_task_data           blob,
   visibility_task_encoding       text,
+  tiered_storage_task_data       blob,
+  tiered_storage_task_encoding   text,
```

Comment:
I think we don't need these 2 new columns for new type of queue. In theory, we just need one pair of data/encoding column for all transfer/replication/visibility/tiered, as long as each of them have different row type.

Reply:
I agree that it is confusing at this moment to have every other queue have their own columns, so I'm not going to push on this. But we should have start with only 1 pair of data/encoding for all queue types.

Reply:
Each of our row type only uses one pair of data/encoding. If we have only 1 pair of data/encoding, all the CQL/SQL queries could be shared as well.

Reply:
we already have different columns for different task types
let us continue the existing path

Reply:
Let's discuss this during the standup tomorrow. It looks like a good candidate for the future refactoring.